src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAAllowEmptyTelemetry.cs

```diff
@@ -8,29 +8,41 @@
 using FluentAssertions;
 using Microsoft.Build.Framework;
 using Microsoft.Build.Tasks;
+using Microsoft.Build.Utilities;
 using Xunit;
 
 namespace Microsoft.NET.Build.Tasks.UnitTests
 {
     public class GivenAAllowEmptyTelemetry
     {
-        [Theory]
-        [InlineData("Property1")]
-        [InlineData("Property1=")]
-        public void WhenInvokeWithoutValueItSendValueAsNull(string eventData)
+        private static ITaskItem CreateHashItem(string key, string value = null, Nullable<bool> hash = null) {
```

Comment:
I would prefer using string? value and #nullable enable, though this is test code so it's not a big deal.

Reply:
However comma, the same could be said about the `var` --> `string?` variables below.

Reply:
me too - I wasn't sure what our position on nullability in tests is in the SDK. happy to do it, though.